<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuListEditor</class>
 <widget class="QWidget" name="MenuListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelImageIcon">
     <property name="text">
      <string>cover</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBottom">
     <property name="text">
      <string>bottom</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDuration">
     <property name="text">
      <string>duration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
